File: app/src/main/java/com/avontell/simcrawl/domain/Event.java
package com.avontell.simcrawl.domain;

import java.util.Date;

/**
 * Represents an Event in Simmons Hall
 * @author Aaron Vontell
 */

public class Event {

    private EventType type;
    private Room room;
    private Date timeStart;
    private Date timeEnd;
    private int count;
    private String title;
    private boolean dormEvent;

    public Event(String title, EventType type, Room room, boolean dormEvent, Date timeStart, Date timeEnd, int count) {
        this.title = title;
        this.type = type;
        this.room = room;
        this.dormEvent = dormEvent;
        this.timeStart = timeStart;
        this.timeEnd = timeEnd;
        this.count = count;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public EventType getType() {
        return type;
    }

    public void setType(EventType type) {
        this.type = type;
    }

    public Room getRoom() {
        return room;
    }

    public void setRoom(Room room) {
        this.room = room;
    }

    public boolean isDormEvent() {
        return dormEvent;
    }

    public void setDormEvent(boolean dormEvent) {
        this.dormEvent = dormEvent;
    }

    public Date getTimeStart() {
        return timeStart;
    }

    public void setTimeStart(Date timeStart) {
        this.timeStart = timeStart;
    }

    public Date getTimeEnd() {
        return timeEnd;
    }

    public void setTimeEnd(Date timeEnd) {
        this.timeEnd = timeEnd;
    }

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }
}

File: python/event.py
class Event:
	def __init__(self,title,host,room,section,dorm_event,start,end,t,count,i):
		self.title = title
		self.host = host
		self.room = room
		self.section = section
		self.dorm_event = dorm_event
		self.start = start
		self.end = end
		self.type = t
		self.count = count
		self.id = i
	
	def get_dict(self):
		return {
			"title":self.title,
			"host":self.host,
			"room":self.room,
			"section":self.section,
			"dormEvent":self.dorm_event,
			"start":self.start,
			"end":self.end,
			"type":self.type,
			"count":self.count,
			"id":self.id
		}	